layer/nrlf/nrlf/core/authoriser.py

```diff
@@ -42,9 +54,12 @@ class Config(BaseModel):
     ENVIRONMENT: str
     SPLUNK_INDEX: str
     SOURCE: str
+    PERMISSIONS_LOOKUP_BUCKET: str
```

Comment:
I think we want to change this to be Authorization store to match the Terraform now

Reply:
Which I should have done when I committed my other PR haha

Reply:
ah yep